### PR TITLE
[DO NOT MERGE] Updating the netcore packages to rc3

### PR DESCRIPTION
**Pull request by livarcocc**
For now, just want to make sure that the latest packages from corefx are green for us.

cc @eerhardt @brthor

### CODE REVIEW

**Comment by livarcocc** [comment]
@dotnet-bot Test this please

File: build_projects/shared-build-targets-utils/Utils/Crossgen.cs [hunk 1]
```diff
@@ -31,41 +31,82 @@ public Crossgen(string coreClrVersion)
...
+
+            string packageId = $"runtime.{coreclrRid}.Microsoft.NETCore.Runtime.CoreCLR";            
+
+            return Path.Combine(
+                Dirs.NuGetPackages,
+                packageId,
+                _coreClrVersion);
+        }
+
+        private string GetCoreCLRRid()
```
Comment (eerhardt) [inline build_projects/shared-build-targets-utils/Utils/Crossgen.cs]: Don't we already have a method for this somewhere?  It is basically "get the current RID", right?

Comment (livarcocc) [inline build_projects/shared-build-targets-utils/Utils/Crossgen.cs]: I was trying to find it, but I wasn't sure where it was and didn't want to get something that would return to me a RID that was similar but not the same. If someone can point me with certainty where this method is, I would LOVE to use it.

**Comment by livarcocc** [comment]
All builds failed in the Kestrel tests.

Kestrel is failing here:

With this error:

Could not load type 'System.Runtime.InteropServices.GCHandle' from assembly 'System.Runtime, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.

File: NuGet.Config [hunk 2]
```diff
@@ -3,6 +3,7 @@
   <packageSources>
     <!--To inherit the global NuGet package sources remove the <clear/> line below -->
     <clear />
+    <add key="dotnet-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
```
Comment (eerhardt) [inline NuGet.Config]: We can remove this now.  All the RC3 packages we need are on cli-deps.

File: build_projects/shared-build-targets-utils/Utils/Crossgen.cs (hunk 1 above)
Comment (brthor) [inline build_projects/shared-build-targets-utils/Utils/Crossgen.cs]: https://github.com/dotnet/cli/blob/rel/1.0.0/build_projects/shared-build-targets-utils/Utils/DotNetCli.cs#L42

**Comment by livarcocc** [comment]
Closing this one in favor of https://github.com/dotnet/cli/pull/3215.

**Pull request closed without merging**